openapi/v2/app-functions-sdk.yml

```diff
@@ -291,23 +271,50 @@ paths:
         content:
           application/json:
             schema:
-              allOf:
-                - $ref: '#/components/schemas/TriggerRequest'
+              type: object
+              description: Object type must match the Application Service's Target Type. Currently Target Type defaults to Edgex model.Events from go-mod-core-contracts. This will change to default to a V2 dtos.Event in the future.
         required: true
       responses:
-        '207':
-          description: "Indicates a multi-part response supportive of accepting multiple requests at once. The 'statusCode' property of each response in the returned array will indicate success or failure."
+        '200':
+          description: "OK"
```

Comment:
We could be a bit more descriptive as to _what_ was successful.

Reply:
What did you have in mind? Doesn't the content description below suffice?